### ISSUE TITLE
Deprecation warning when using ActiveModel::SerializableResource.new does not specify alternative

### ISSUE THREAD

**Issue by eric-norcross** [rails-api/active_model_serializers#2280]
Message: 
NOTE: ActiveModel::SerializableResource.new is deprecated; use ActiveModelSerializers::SerializableResource. instead

#### Expected behavior vs actual behavior
Expected behavior: The deprecation message should Indicate what the new/replacement method is.
Actual behavior: The new method is omitted

#### Steps to reproduce
Call `ActiveModel::SerializableResource.new`

#### Environment
ActiveModelSerializers Version: 
0.10.7

Output of `ruby -e "puts RUBY_DESCRIPTION"`: 
ruby 2.5.0p0 (2017-12-25 revision 61468) [x86_64-darwin16]

OS Type & Version: 
MacOS 10.12.6

Integrated application and version *(e.g., Rails, Grape, etc)*: 
Rails 5.1.4

**Comment by juliolugo96** [comment]
Sorry for delayed response. I hope this could help someone who has this same problem.

As far as I concerned, I used `initialize()` method; however, I think you can use the `new` method too, if I'm not misintrepreting what it's pointed in the[ docs ](https://www.rubydoc.info/gems/active_model_serializers/ActiveModelSerializers/SerializableResource).

Hope it helps. Cheers!

**Comment by bf4** [comment]
yeah, it should read `ActiveModelSerializers::SerializableResource.new`

@juliolugo96 `new` is a class method `initialize` is an instance method (called by the class method `new` on the return value of the class method `allocate`)

**Comment by juliolugo96** [comment]
@bf4 That's right. Thanks for the clarification, I'll update my comment.
